cmd/nginx-ingress/aws.go

```diff
@@ -1,4 +1,4 @@
-// +build aws
+//go:build aws
```

Comment:
which tool uses this tag? I'm not familiar with it

Reply:
So you can do something like `go build -tags=aws github.com/nginxinc/kubernetes-ingress/cmd/nginx-ingress` (and GoReleaser does the same)